### PR TITLE
feat(server): do not automatically download android motion videos

### LINKED ISSUE #11742: Download returns image AND video for images with embedded motion video
### The bug

I have a few JPGs that contain motion videos as created by certain Samsung phones. These short videos show up as part of the image when viewed in immich. When downloading such an image, this results in two downloaded files: the actual image (which still includes the video), and the (extracted) video. When downloading several images, this issue also exists in the resulting ZIP file.

This wastes disk space, causes longer downloads and also causes duplicates when re-importing using the data from the downloads.

I suggest to NOT add the video to the download file, just adding the photo should be enough.

### The OS that Immich Server is running on

Debian Stable

### Version of Immich Server

v1.111.0

### Version of Immich Mobile App

none

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

defaults

### Your .env content

defaults

### Reproduction steps

1. Have image asset with embedded video (motion video, Samsung phones)
2. Download image

### Relevant log output

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jrasm91**
Fixes #11742

- Skip downloading or including in a download, extracted android motion videos
- Continue to include motion videos from iOS (not embedded)
- Add a user preference (`download.includeEmbeddedVideos`) to change this behavior (per user)
- Use originalPath in `encoded-videos` folder to determine if it is extracted or not

image

### CODE REVIEW

**Comment by michelheusschen** [comment]
iOS live photos are made up of a separate photo and video part, while android motion photos are contained within a single file. It seems a bit unusual to even have the option to download android motion photos as two separate files. Is there a use case behind that?

**Comment by C-Otto** [comment]
> Is there a use case behind that?

I'd say no. Having the video in the downloaded JPG is good enough, I can always split it when I need to.

**Comment by jrasm91** [comment]
There are plenty of people searching online trying to figure out how to extract the mp4 video. Usually the answer is run something like a bash script looking for a magic byte, etc. I assumed users would like the option to be able to download the video directly from immich to view it, rather than rely having to do it out of band.

**Review by danieldietzler — APPROVED** [review]
LGTM

File: server/src/dtos/user-preferences.dto.ts [hunk 1]
```diff
@@ -103,6 +106,8 @@ class EmailNotificationsResponse {
 class DownloadResponse {
   @ApiProperty({ type: 'integer' })
   archiveSize!: number;
+
+  includeEmbeddedVideos: boolean = false;
```
Comment (alextran1502) [inline server/src/dtos/user-preferences.dto.ts:110]: Do we need to patch this in mobile client?

Comment (jrasm91) [reply server/src/dtos/user-preferences.dto.ts:110]: I don't think so. It is a new property and it has a default value.

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**